svg/elements/discard.json

```diff
@@ -26,22 +27,25 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "21",
+              "version_removed": "68"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "21",
+              "version_removed": "58"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0",
+              "version_removed": false
```

Comment:
You can omit the non-removal:

```suggestion
```

Reply:
Will do, but shouldn't this fail the lint? :)